README.md

```diff
@@ -37,7 +37,9 @@ You can install it on Foreman 2.5 to prepare for the Foreman update.
 
 |Foreman version|Plugin version|Notes                                     |
 |---------------|--------------|------------------------------------------|
-| >= 3.0        | ~> 1.0       | Required                                 |
+| >= 3.2        | ~> 3.0       | Required                                 |
```

Comment:
not sure if we won't have 2.1 instead (unless we have breaking changes.
But I guess it doesn't matter, we can just release 3.0 :shrug:

Reply:
I kind of assumed it will be 3.0 as there is a 3.0 tag already. In any case, we can adjust the matrix when we get there.